packages/notifications/push/amplify_push_notifications/android/src/main/kotlin/com/amazonaws/amplify/amplify_push_notifications/AmplifyPushNotificationsPlugin.kt

```diff
@@ -60,53 +66,54 @@ open class AmplifyPushNotificationsPlugin : FlutterPlugin, ActivityAware,
      */
     private var mainBinaryMessenger: BinaryMessenger? = null
 
-    /**
-     * Flutter API interface.
-     */
-    private var flutterApi: PushNotificationsFlutterApi? = null
 
     /**
      * Launch notification has the notification map when app was launched by tapping on the notification
      * when the app is in killed state and null otherwise.
      */
     private var launchNotification: MutableMap<Any, Any?>? = null
 
+
     override fun onAttachedToEngine(flutterPluginBinding: FlutterPlugin.FlutterPluginBinding) {
         mainBinaryMessenger = flutterPluginBinding.binaryMessenger
```

Comment:
Looking at this file, this assignment used only in this method implementation, can make it local, so you don't need `!!` and reset it as null below.

Reply:
`launchNotification` needs to be nullable so we can assign it null after it's consumed.